### ISSUE TITLE
Deluge rpc setup isn't clear 

### ISSUE THREAD

**Issue by OfryL** [erickok/transdroid#440]
I tried to used it many times. It worked only after reading the pull request that make me enable the 'accept ssl from all'. It's sad that I tried to configure it so many times before and had to read the git to work it out.

**Comment by erickok** [comment]
Yes a guide would be nice to have. I would not recomment using the Accept all certificates option as it is inherently insecure. Of course for local connections it might be just fine though.

**Comment by uzsito** [comment]
Yeah, I had to struggle a few hours with it too, before ended up here. :)
A quick setup guide would be nice indeed.

**Comment by ookami125** [comment]
As I was just having trouble finding how to get a Sha1 of the certificate and I was lead here multiple times I would like to include that
`echo -n | openssl x509 -noout -fingerprint -sha1 -inform pem -in daemon.cert`
is a valid way of getting the sha1 fingerprint from the file and 
`echo -n | openssl s_client -showcerts -connect HOST:PORT | openssl x509 -fingerprint -noout -sha1 -inform pem` **(insecure)**
will pull it from a remote server. Those 2 commands should work in both linux and windows as long as openssl is installed (tested in Command Prompt and WSL Ubuntu Bash).

**Comment by Antoniossss** [comment]
So how to actually make deluge RPC work in transdroid? Windows client works so I guess transdroid should be able to connect as well, but I have no clue how to do proper config.

**Comment by Sanaki** [comment]
When you say the windows client works, I assume you mean on the same system as the server. The auth file authorizes localclient by default, so [you'll need to add an entry for the user you want to use in transdrone](https://dev.deluge-torrent.org/wiki/UserGuide/Authentication). You'll also need to enable remote connections in the daemon section of your preferences. Past that it's pretty straightforward. Do use SSL, and get the certificate fingerprint via the methods stated above.

**Comment by Antoniossss** [comment]
Your asumption is wrong, I use remote server and windows client connects to it without any problems.

**Comment by Sanaki** [comment]
In that case, you should be able to connect just fine as long as you get the SSL fingerprint stored and have a user for the connection.

**Comment by JuniorJPDJ** [comment]
Thank you very much for this issue, it helped me so much :X

**Comment by brainchild0** [comment]
I would never have succeeded in connecting to my Deluge daemon without finding this thread. I had already connected without problems without from the official GTK desktop client. Support for greater security is useful, but most deployments occur inside of small networks accessible to a small group of trusted individuals.

 Imposing a more rigid security model in Transdroid  than in the desktop application, without some kind of explanation inside the application's menu structure, creates a barrier for use.
